zebrad/tests/acceptance.rs

```diff
@@ -1212,6 +1212,51 @@ async fn rpc_endpoint() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn non_blocking_logger() -> Result<()> {
+    let _init_guard = zebra_test::init();
+
+    // Write a configuration that has RPC listen_addr set
+    // [Note on port conflict](#Note on port conflict)
+    let mut config = random_known_rpc_port_config()?;
+    let zebra_rpc_address = config.rpc.listen_addr.unwrap();
+
+    let dir = testdir()?.with_config(&mut config)?;
+    let mut child = dir.spawn_child(args!["start"])?;
+
+    // Wait until port is open.
+    child.expect_stdout_line_matches(
+        format!("Opened RPC endpoint at {}", config.rpc.listen_addr.unwrap()).as_str(),
+    )?;
+
+    // Create an http client
+    let client = reqwest::Client::new();
+
+    for _ in 0..10_000 {
```

Comment:
Is this enough to reach the default 128_000 line limit?

Reply:
Definitely not, this is to test that there isn't a deadlock when there are a lot of queries in quick succession